lxd/main.go

```diff
@@ -633,6 +635,10 @@ func setupLXD() error {
 	}
 
 	if *argAuto {
+		if *argStorageBackend == "" {
```

Comment:
If you want to change the config after intiial configuration, that's what "lxc config set" is for.